manifests/git_repo.pp

```diff
@@ -9,7 +9,7 @@
   Boolean $use_ssh_fork = $katello_devel::use_ssh_fork,
 ) {
 
-  if $github_username {
+  if $github_username != undef or $github_username != '' {
```

Comment:
Shouldn't that be `and` because this is always true?

Reply:
Good call -- updated